backend/API_standards.md

```diff
@@ -52,14 +52,32 @@ Nested resources will follow the same pattern:
 
 All endpoints should return a seamless response, independent of the endpoint action or expected result values.
 
+### Sessions
+
+To perform Login should be `POST /sessions` with email and password and it will returns an object with the auth information (token, expire date, etc) and the user information, being same User entity as in the others endpoints, everything inside of the payload key `data`.
+
+Response example:
+
+```json
+{
+    "data": {
```

Comment:
Why inside a "data" ?

Reply:
+1 to @jfelipe comment, you can see more details here http://jsonapi.org/format/#document-top-level

Also, this was a suggestion from one of our engineers (Daniel Gomez) [see here](https://github.com/BaristaVentures/craftsman-hand-book/pull/18#issuecomment-305246030) 😅

Reply:
@jfelipe @orendon Im asking because this is for every endpoint, I was talking there just about paginated endpoints but I dont know anything about jsonapi. Im just imaging myself making wrappers for every endpoint and feeling some pain on the process :P

Reply:
But thinking again, it may not be horrible :)

Reply:
Yes, all responses would have a payload identifier, what this does is standardize that identifier name to `data`.

I believe that is actually easier than having a different payloads identifier on every endpoint (i.e user, questions, question, questions_per_user, and so on...)

Reply:
@caipivara I think is better to keep the same standard for all resources.